src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/Modules/RoModule.cs

```diff
@@ -30,11 +31,21 @@ internal RoModule(string fullyQualifiedName)
         public sealed override Assembly Assembly => GetRoAssembly();
         internal abstract RoAssembly GetRoAssembly();
 
+
+
+        internal const string UnknownStringMessageInRAF = "Returns <unknown> for modules with no file path";
+
+#if NETCOREAPP
```

Comment:
Seems needs `#if NET5_0_OR_GREATER`
```suggestion
#if NET5_0_OR_GREATER
```